pkgs/memory_usage/analysis_options.yaml

```diff
@@ -0,0 +1,143 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Since this is not a flutter package, I would highly recommend moving to either package:lints/recommended.yaml (what we recommend to dart users) or package:dart_flutter_team_lints/analysis_options.yaml (what we use for many package in dart_lang). The 2nd will be more restrictive, so the 1st may be easier to move to. Either one will replace the contents of this file w/ a simple include which will be a nice benefit.

Reply:
For now I am copy-pasting this across packages. Created issue to fix it: https://github.com/dart-lang/leak_tracker/issues/175

Thanks.